examples/phi2/README.md

```diff
@@ -35,7 +53,7 @@ Above commands will generate optimized models with given model_type and save the
 Besides, for better generation experience, this example also let use use [Optimum](https://huggingface.co/docs/optimum/v1.2.1/en/onnxruntime/modeling_ort) to generate optimized models.
 Then use can call `model.generate` easily to run inference with the optimized model.
 ```bash
-# optimum optimization
```

Comment:
If we don't support it, should we remove this?

Reply:
For the model that did not be fine tuned, optimum works well.